syz-manager/manager.go

```diff
@@ -592,18 +592,24 @@ func (mgr *Manager) isSuppressed(crash *Crash) bool {
 	return false
 }
 
-func (mgr *Manager) emailCrash(crash *Crash) {
-	if mgr.cfg.Email_Addr == "" {
+func (mgr *Manager) emailTo(crash *Crash, addr string) {
+	if addr == "" {
 		return
 	}
-	Logf(0, "sending email to %v", mgr.cfg.Email_Addr)
-	cmd := exec.Command("mailx", "-s", "syzkaller: "+crash.Title, mgr.cfg.Email_Addr)
+	Logf(0, "sending email to %v", addr)
+	cmd := exec.Command("mailx", "-s", "syzkaller: "+crash.Title, addr)
```

Comment:
mailx supports sending a single email to multiple recipients, so I think it would be better to send a single email (faster, same Message-ID):

```go
if len(mgr.cfg.Email_Addrs) == 0 {
  return
}
...
args := []string{"-s", "syzkaller: "+crash.Title}
args = append(args, mgr.cfg.Email_Addrs...)
cmd := exec.Command("mailx", args...)
```

Reply:
Good call. People might want to see who else is getting this bug report because it might contain sensitive info.